app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -29,47 +28,52 @@ def manage
 
   # POST /wrangling_guidelines
   def create
-    @wrangling_guideline = WranglingGuideline.new(params[:wrangling_guideline])
+    @wrangling_guideline = WranglingGuideline.new(wrangling_guideline_params)
 
     if @wrangling_guideline.save
       flash[:notice] = ts('Wrangling Guideline was successfully created.')
       redirect_to(@wrangling_guideline)
     else
-      render :action => 'new'
+      render action: 'new'
     end
   end
 
   # PUT /wrangling_guidelines/1
   def update
-    @wrangling_guideline = WranglingGuideline.find(params[:id])
+    @wrangling_guideline = WranglingGuideline.find(wrangling_guideline_id)
 
-    if @wrangling_guideline.update_attributes(params[:wrangling_guideline])
+    if @wrangling_guideline.update_attributes(wrangling_guideline_params)
       flash[:notice] = ts('Wrangling Guideline was successfully updated.')
       redirect_to(@wrangling_guideline)
     else
-      render :action => 'edit'
+      render action: 'edit'
     end
   end
 
   # reorder FAQs
   def update_positions
     if params[:wrangling_guidelines]
-      @wrangling_guidelines = WranglingGuideline.reorder(params[:wrangling_guidelines])       
+      @wrangling_guidelines = WranglingGuideline.reorder(params.permit(:utf8, :commit, wrangling_guidelines: [])[:wrangling_guidelines])
```

Comment:
But we are only using wrangling_guidelines I could remove the other bits but I think it would be counter intuitive. The raising of the error only happens in test. In production strong_params will get rid of params we do not permit.